dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/DeletedUserHelper.java

```diff
@@ -0,0 +1,30 @@
+package io.dockstore.webservice.helpers;
+
+import java.sql.Timestamp;
+import java.time.Duration;
+import java.time.LocalDateTime;
+
+import io.dockstore.webservice.core.DeletedUsername;
+import io.dockstore.webservice.jdbi.DeletedUsernameDAO;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class DeletedUserHelper {
```

Comment:
Is it worth having this class now? Now that you moved the date logic into the query? I lean towards it not being necessary, but I don't feel super strongly about it.

Reply:
I guess it's not necessary, but I kept it since since the function is used 4 times across 2 files and time stamp needs to be calculated each time. I could move the function somewhere else, but I just kept it here.